tests/test_bad_fonts_nonrecursive.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
I don't like it either, but unfortunately the existing script already takes an argument, and autoconf doesn't allow us to pass arguments to test scripts anyway. I suppose the way to handle this without code duplication would be to modify the interface of test_bad_fonts.sh and then have test_bad_fonts_nonrecursive.sh call test_bad_fonts.sh.